JavaScript/JavaScript.sublime-syntax

```diff
@@ -207,13 +207,13 @@ contexts:
         8: storage.type.function.arrow.js
     - match: |-
         (?x)
-        (\b_?[A-Z][$\w]*)?
+        (\b_?[a-zA-Z_?.$]*)?
```

Comment:
Alright, for now let's use `(\b_?[a-zA-Z_.$]*)?`.

I need to spend some time today on JavaScript.sublime-syntax since we are seeing indexing performance issues with the new additions in 3104. So there may end up being some significant changes upcoming.